DependencyInjection/Configuration.php

```diff
@@ -76,11 +74,14 @@ public function getConfigTreeBuilder()
                     ->normalizeKeys(false)
                     ->prototype('array')
                         ->fixXmlConfig('extra')
+                        ->fixXmlConfig('stacked_processor', 'stacked_processors')
```

Comment:
You only need the first argument as the plural form is the regular one (just a "s" at the end).